lib/store_base_sti_class_for_5_0.rb

```diff
@@ -346,6 +346,24 @@ def build_through_record(record)
     end
 
     module Reflection
+      class PolymorphicReflection < ThroughReflection
```

Comment:
@n-rodriguez please disregard outdated comments and simply remove the  ``` < ThroughReflection```
tested.